src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -311,7 +146,7 @@ function LoadoutBuilder({
       assumeMasterwork,
     },
     stateDispatch,
-  ] = useReducer(stateReducer, { stores, location }, init);
+  ] = useReducer(lbStateReducer, { stores, location }, lbStateInit);
```

Comment:
Rather than importing these two, you could make a `useLbState` hook that just calls `useReducer`. Either way, doesn't really matter.

Reply:
Yeah I like that, I will sort it out tonight.